eos/rare-b-decays/b-to-vec-nu-nu.cc

```diff
@@ -173,27 +173,43 @@ namespace eos
         {
             const double m_B = this->m_B(), m_B2 = m_B * m_B;
             const double m_V = this->m_V(), m_V2 = m_V * m_V;
+            const double m_b = model->m_b_msbar(mu);
+            const double m_s = model->m_s_msbar(mu);
```

Comment:
Can you please create a new member of the Implementation class, ``std::function<double (const double &)> m_D`` and assign it a lambda returning ``model->m_s_msbar(mu);``, between lines 160 and 161. Makes it easier if we want to implement ``dbnunu`` decays in the future :smile:

Reply:
Good idea. I will rename `m_s` to `m_D` everywhere in the equation implementation as well. I will add this in a new commit and we can squash later when we are happy.